src/lib.rs

```diff
@@ -590,8 +590,8 @@ macro_rules! __pin_project_internal {
 
             // Ensure that it's impossible to use pin projections on a #[repr(packed)] struct.
             //
-            // Taking a reference to a packed field is unsafe, amd appplying
-            // #[forbid(safe_packed_borrows)] makes sure that doing this without
+            // Taking a reference to a packed field is unsafe, and appplying
```

Comment:
Making it unsafe was a temporary hack but didn't really make sense. In the future, this is going to be just forbidden.